robosat/log.py

```diff
@@ -0,0 +1,26 @@
+"""Log facilitator
+"""
+
+import os
+
+
+class Log:
+
+    """Create a log instance on a log file
+    """
+
+    def __init__(self, path, stdout=True):
+        self.stdout = stdout
+        self.fp = open(path, "a")
```

Comment:
Should we `.close` the file descriptor explicitly somewhere?

Reply:
Hm, if we flush it should be fine I guess. Was just wondering if we can expose this in a clean way.